pkg/kubelet/cm/cpumanager/policy_options_test.go

```diff
@@ -168,10 +185,13 @@ func TestValidateStaticPolicyOptions(t *testing.T) {
 			topoMgrStore := topologymanager.NewFakeManagerWithPolicy(topoMgrPolicy)
 
 			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, pkgfeatures.CPUManagerPolicyAlphaOptions, true)()
-			policyOpt, _ := NewStaticPolicyOptions(testCase.policyOption)
-			err := ValidateStaticPolicyOptions(policyOpt, testCase.topology, topoMgrStore)
+			policyOpt, err := NewStaticPolicyOptions(testCase.policyOption)
+			if err == nil && testCase.expectedErr {
```

Comment:
This check misses the case with ```expectedErr = false``` and ```err != nil```. Can you use the same approach as with ```expectedValidateErr``` below?

Reply:
Sure thing, done in 1578af56b87